### ISSUE TITLE
[Northrend] [Quests] Generosity Abounds, Matchmaker, Stunning View

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3901]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
 During the quest Generosity Abounds, sometimes the mines carried glitch out and aren't visible. I found out that only those on the floor near the forge do that.
 Also, after dragging mines for the quest Generosity Abounds, if the Lithe Stalker uses any abilities, he'll be forced to walk by default, disabling him from running. That ofc can be fixed by pressing the walk/run button.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 Some bombs become invisible, and the Stalker is forced to walk after using skills.

**Expected behaviour**: Tell us what should happen instead
 Bombs shouldn't become invisible, and the Stalker should be able to run after using skills without being forced to walk.

**Steps to reproduce the problem**:

1.  Take the quests.
2.  Pass through them.
3. 

**Include proofs for this behaviour**

https://postimg.org/gallery/x3m3xi6y/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
> if the Lithe Stalker uses any abilities, he'll be forced to walk by default, disabling him from running. 

generic issue for all vehicles.

i will look into this issue:
> During the quest Generosity Abounds, sometimes the mines carried glitch out and aren't visible. I found out that only those on the floor near the forge do that.

**Comment by Rushor** [comment]
DB/Quest: Generosity Abounds - Bombs should always be visible

* closes https://github.com/dalaranwow/dalaran-wow/issues/3901
* commit: 23e4535939c5fb0e29afcb2921b65a2de7d4e509

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
issue:

> Also, after dragging mines for the quest Generosity Abounds, if the Lithe Stalker uses any abilities, he'll be forced to walk by default, disabling him from running. That ofc can be fixed by pressing the walk/run button.

will be handled in https://github.com/dalaranwow/dalaran-wow/issues/3904

**Comment by TeridanDW** [comment]
Works as intended on PTR.
